server/http/api_v2.go

```diff
@@ -309,7 +310,13 @@ func (api *APIv2) ServeRead(w http.ResponseWriter, r *http.Request) {
 		jsonResponse(w, http.StatusBadRequest, err)
 		return
 	}
-	qr := graph.NewQuadStoreReader(h.QuadStore)
+	sub := gephi.ValuesFromString(r.FormValue("sub"))
```

Comment:
Can you please update `./docs/api/swagger.yml` as well, so it lists new parameters? You can use https://editor.swagger.io/ to validate it.